scripts/DGSPlanner/DGSPlannerGUI.py

```diff
@@ -10,6 +10,7 @@
 import mantidqtpython as mqt
 from .ValidateOL import ValidateOL
 from matplotlib.backends.backend_qt4agg import FigureCanvasQTAgg as FigureCanvas
+from matplotlib.backends.backend_qt4agg import NavigationToolbar2QTAgg as NavigationToolbar
```

Comment:
This fails to run on analysis or on my computer It should be `NavigationToolbar2QT` not `NavigationToolbar2QTAgg`

Reply:
OK, this actually depends on the version.  I used on my home machine.  Thank you for changing.